### ISSUE TITLE
Determining conformance

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wai-intro-wcag#102]
In WCAG intro we currently say:

> The success criteria are what determine “conformance” to WCAG. That is, in order to meet WCAG, the content needs to meet the success criteria.

What determines conformance in WCAG2 is [Conformance Requirements](https://www.w3.org/TR/WCAG21/#conformance)

Which of course comprises SCs but implies more than these: complete processes, accessibility support, no interference, etc.

**Comment by shawna-slh** [comment]
I can't find this sentence -- perhaps it's already changed. Feel free to re-open if you find it.

**Comment by shawna-slh** [comment]
oh-- I found it in a preview... so will address...

**Comment by shawna-slh** [comment]
Hi @daniel-montalvo
We want to keep this point simple. How about if we add a link to the Conformance section:
> The success criteria are what determine "conformance" to WCAG. That is, in order to meet WCAG, the content needs to meet the success criteria. Details are in the [Conformance section of WCAG](https://www.w3.org/TR/WCAG21/#conformance).

**Comment by daniel-montalvo** [comment]
Thanks for coming back to me on this, @shawna-slh 

> We want to keep this point simple. How about if we add a link to the Conformance section:
>> The success criteria are what determine "conformance" to WCAG. That is, in order to meet WCAG, the content needs to meet the success criteria. Details are in the [Conformance section of WCAG](https://www.w3.org/TR/WCAG21/#conformance).

IMO, that still conveys that the only thing you have to do is meet the SCs. There is a lot more in that section.

- Full Pages
- Complete Processes
- Accessibility Support
- Non-interference

I am concerned because we are missing the "accessibility support" bit when talking about conformance. What if you meet an SC in a way that is not usable by people with disabilities?

So maybe something like:

> The success criteria are what determine "conformance" to WCAG. That is, in order to meet WCAG, the content needs to meet the success criteria and be usable by people with disabilities. Additional conformance requirements are in the [Conformance section of WCAG](https://www.w3.org/TR/WCAG21/#conformance).

This sets the idea that conformance is based on meeting SCs in an accessibility supported way. Caveat: we need to look for workarounds to not mention "accessibility support" which is such a complex term. Maybe "be usable by people with disabilities" can be easily misunderstood.

**Comment by shawna-slh** [comment]
Thanks for the additional input. I will take another pass at it!

**Comment by akasyagroup** [comment]
WCAG2&#x<>;de Uygundur
